amazonia/classes/asg.py

```diff
@@ -111,13 +115,14 @@ def create_asg(self, title, minsize, maxsize, subnets, load_balancer, keypair, i
             image_id=image_id,
             instance_type=instance_type,
             iam_instance_profile_arn=iam_instance_profile_arn,
-            userdata=userdata
+            userdata=userdata,
+            hdd_size=hdd_size
         ))
         if userdata is None:
             self.lc.UserData = ''
         return title
 
-    def create_launch_config(self, title, keypair, image_id, instance_type, iam_instance_profile_arn, userdata):
+    def create_launch_config(self, title, keypair, image_id, instance_type, iam_instance_profile_arn, userdata, hdd_size):
```

Comment:
Update
add http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-launchconfig.html
and add http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-launchconfig-blockdev-mapping.html